ARMDeploy/ohbooking.json

```diff
@@ -46,6 +46,7 @@
         "serviceBusTopicNotificationsName": "mailnotifications",
         "serviceBusTopicUpdatesName": "mailupdates",
         "serviceBusQueueNotificationResponsesName": "mailidresponses",
+        "serviceBusQueueBookingSlot": "newbookingslot",
```

Comment:
A placeholder has been created in Outlook, Flow has sent the start and end time over a queue called 'newBookingSlot', a function listens to the 'newBookingSlot' queue and creates a booking slot. More than willing to change it, any suggestions?